SupersonicSound/LowLevel/LowLevelSystem.cs

```diff
@@ -221,11 +221,19 @@ void IPreInitilizeLowLevelSystem.SetCallback(Action<LowLevelSystem, SystemCallba
         #endregion
 
         #region init/close
+        [Obsolete("Not sure why this method is in here, the system should already be initialized when passed in through the constructor")]
         public void Initialize(int maxchannels, LowLevelInitFlags flags)
         {
             _system.init(maxchannels, (INITFLAGS) flags, IntPtr.Zero).Check();
         }
 
+        [Obsolete("This should only be called when we've initialized a low level system without studio (I think)")]
```

Comment:
Ah I see, this prevents the disposal of the low level system contained within the studio system. I think a better way would be to have an internal method which sets a flag on the low level system indicating that it is contained within a parent. Then inside the dispose method we can check the flag.

This means you can still use using(LowLevelSystem) and things work properly.